ad9361/sw/ad9361.h

```diff
@@ -3541,4 +3541,5 @@ int32_t ad9361_get_temp(struct ad9361_rf_phy *phy);
 int ad9361_synth_lo_powerdown(struct ad9361_rf_phy *phy,
```

Comment:
there is a typo in the commit message ad9361 instead of d9361. The message is vague, it should contain more information about what it does.